docs/modules/execa.md

```diff
@@ -0,0 +1,13 @@
+# Execa
+
+When you don't need [additional features](https://github.com/sindresorhus/execa/#features) that `execa` provides such as [string templating](https://github.com/sindresorhus/execa/blob/main/docs/execution.md#template-string-syntax), [inter-process communication](https://github.com/sindresorhus/execa/#exchange-messages), and [debugging features](https://github.com/sindresorhus/execa/?tab=readme-ov-file#detailed-error),  you can replace it with tinier alternatives below.
```

Comment:
> execa and other execution libraries have lighter alternatives available, especially if you do not need a higher level abstraction (e.g. templating syntax, scripting, etc).

something as simple as this would be fine as this doc won't always be about execa

we don't need to explain why someone should use execa here. this doc is here to explain why one would move off execa and other libraries.

it also needs renaming i think to `process-exec.md` (same as your links in the `README`)

Reply:
Rewritten based on your feedbacks